packages/matchbox/src/components/Columns/Columns.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import propTypes from '@styled-system/prop-types';
+import styled from 'styled-components';
+import { pick } from '@styled-system/props';
+import { createPropTypes } from '@styled-system/prop-types';
+import { margin } from 'styled-system';
+import { tokens } from '@sparkpost/design-tokens';
+import { ColumnsContext } from './context';
+import { verticalAlignment, horizontalAlignment, reverseColumns, negativeMargin } from './styles';
+import { useWindowSize } from '../../helpers/geometry';
+import { Box } from '../Box';
+
+const breakpoints = ['xs', 'sm', 'md', 'lg', 'xl'];
+
+const StyledColumns = styled(Box)`
+  ${verticalAlignment}
+  ${horizontalAlignment}
+  ${reverseColumns}
+  ${negativeMargin}
+`;
+
+const Columns = React.forwardRef(function Columns(props, ref) {
+  const { children, reverse, space, alignY, align, collapseBelow, ...rest } = props;
+  const [collapsed, setCollapsed] = React.useState(false);
+
+  const systemProps = pick(rest);
```

Comment:
Could you use the other `pick` helper with `pick(rest, margin.propNames)` from helpers/systemProps? Since this is being spread onto a `Box`, all the other props would work